### PR TITLE
Web 22 footer section

### PR DESCRIPTION
**Pull request by genebarker**
# Describe your changes

Add initial version of footer section.

## Issue ticket number and link

WEB-22

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

File: src/lib/components/footer.svelte [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            Jira
+        </a>
+        &nbsp;&nbsp;
+        <a
+            class="btn btn-sm variant-ghost-surface"
+            href="https://samoa-stack-overflow.atlassian.net/wiki/spaces/SW/overview"
+            target="_blank"
+        >
+            Confluence
+        </a>
```
Comment (westerandr) [inline src/lib/components/footer.svelte:29]: Can we remove this please? and Add Facebook group link.

Here it is: https://www.facebook.com/groups/948415479224570

File: src/lib/components/footer.svelte [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    </svelte:fragment>
+    <svelte:fragment slot="trail">
+        Copyright © 2023 Samoa Stack Overflow. All Rights Reserved.
```
Comment (westerandr) [inline src/lib/components/footer.svelte:32]: Can we make the year dynamic please?

new Date().getYear() should do the trick

File: src/lib/components/footer.svelte [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+            GitHub
+        </a>
+        &nbsp;&nbsp;
+        <a
+            class="btn btn-sm variant-ghost-surface"
+            href="https://samoa-stack-overflow.atlassian.net/browse/WEB"
+            target="_blank"
+        >
+            Jira
+        </a>
```
Comment (westerandr) [inline src/lib/components/footer.svelte:21]: Can the link and title be changed. I wouldn't want the public to see our JIRA board and tickets lol

Comment (westerandr) [reply src/lib/components/footer.svelte:21]: Instead you can use Discord as title and link can be # for now, while I sort out a survey that people have to answer before getting sent a link to the discord.

**Review by westerandr — CHANGES_REQUESTED** [review]

File: src/lib/components/footer.svelte (hunk 3 above)
Comment (westerandr) [reply src/lib/components/footer.svelte:21]: or use GitHub i.e. https://github.com/Samoa-Stack-Overflow

File: src/lib/components/footer.svelte (hunk 1 above)
Comment (genebarker) [reply src/lib/components/footer.svelte]: Done. See latest :-)

File: src/lib/components/footer.svelte (hunk 2 above)
Comment (genebarker) [reply src/lib/components/footer.svelte]: Done. See latest.

File: src/lib/components/footer.svelte (hunk 3 above)
Comment (genebarker) [reply src/lib/components/footer.svelte]: Done. See latest commit.

**Review by westerandr — APPROVED** [review]
LGTM!

**Pull request merged**